.github/workflows/functional-test-cloud.yaml

```diff
@@ -348,10 +350,49 @@ jobs:
           append: true
           message: |
             :x: Test recipe publishing failed
+  
+  publish-test-bicep-types:
```

Comment:
Should this be moved inside the build job?

Reply:
I kept it as a separate job so it doesn't make the build step too long and can run in parallel. I'm not opposed to moving it into the build step though if that'd be better for this workflow